### PR TITLE
Add note about maximum latency.

### PR DESCRIPTION
**Pull request by mfoltzgoogle**
Addresses Issue #80 by adding an implementation note about message playback and placing an upper bound on message handling latency.

Basically for latency, "the lower the better," but this uses the ITU recommendation as an upper bound.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/openscreenprotocol/pull/319.html" title="Last updated on Oct 16, 2023, 5:17 PM UTC (0cdd969)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/openscreenprotocol/319/ee2b93f...0cdd969.html" title="Last updated on Oct 16, 2023, 5:17 PM UTC (0cdd969)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -886,6 +886,12 @@ To send a presentation message, the controller or receiver may send a
 : message
 :: The presentation message data.
 
+NOTE: An OSP agent should minimize the latency between applications sending or
+receiving messages and the underlying QUIC connection, as messages may be used
```
Comment (baylesj) [inline index.bs:890]: Is this specifically referring to having performant message serialization and deserialization? It's unclear to me what exactly the steps are between sending a message and it reaching the QUIC API for message sending.

**Comment by baylesj** [comment]
LGTM % feedback

**Review by baylesj — APPROVED** [review]
LGTM % feedback.

**Review by baylesj — APPROVED** [review]
LGTM % feedback.

File: index.bs (hunk 1 above)
Comment (mfoltzgoogle) [reply index.bs:890]: Serialization is one necessary step; but basically it's telling the implementation that messages need to be as "real-time" as possible, so don't add unnecessary message buffering in the OSP implementation on top of network buffering in QUIC.

I will add some more detail here along these lines.

**Comment by mfoltzgoogle** [comment]
PTAL baylesj@

**Review by chrisn — APPROVED** [review]

**Review by baylesj — APPROVED** [review]
LGTM.

**Pull request merged**